### PR TITLE
[bitnami/testlink] adds feature for creating database/user on external instance

### PR DESCRIPTION
**Pull request by steffancarlo**
**Description of the change**
Currently when not using the MariaDB chart, you can only point to an existing external DB/user. This PR will introduce a new option to let the mysql-client create the DB and user for you.

**Benefits**
Allows users to decide whether to use an existing DB/user or to create a new one on the external instance.

**Possible drawbacks**
NA

**Applicable issues**
NA

**Additional information**
NA

**Checklist**
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

### CODE REVIEW

**Comment by javsalgar** [comment]
Hi! Thanks for the PR! Could you bump the minor version instead? It is adding a non-breaking feature.

**Comment by javsalgar** [comment]
LGTM, thanks! I will trigger our automatic build system to update the tags.

File: bitnami/testlink/values.yaml [hunk 1]
```diff
@@ -71,6 +71,18 @@ allowEmptyPassword: "yes"
...
+  ##If set to 'true' db and db user will be created as specified
+  create: false
+  
+  ## Database user privileges
+#  privileges: ALL
+
+  ##admin user to create db and user.
+#  root:
+#    user:
+#    password:
```
Comment (javsalgar) [inline bitnami/testlink/values.yaml:84]: Our automated testing system is failing with this structure. Do you mind changing it to rootUser and rootPassword?

Comment (steffancarlo) [reply bitnami/testlink/values.yaml]: No problem, changes done.

**Pull request merged**